core/templates/pages/contributor-dashboard-page/login-required-message/login-required-message.component.ts

```diff
@@ -20,18 +20,22 @@ import { SiteAnalyticsService } from 'services/site-analytics.service';
 import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
 import { UserService } from 'services/user.service';
 import { WindowRef } from 'services/contextual/window-ref.service';
-import constants from 'assets/constants';
 import { Component } from '@angular/core';
 import { downgradeComponent } from '@angular/upgrade/static';
+import { AppConstants } from 'app.constants';
 
 @Component({
   selector: 'login-required-message',
   templateUrl: './login-required-message.component.html',
   styleUrls: []
 })
 export class LoginRequiredMessageComponent {
-  OPPIA_AVATAR_IMAGE_URL: string;
-  OPPIA_AVATAR_LINK_URL: string;
+  // These properties are initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion, for more information see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1 .
+  OPPIA_AVATAR_IMAGE_URL!: string;
+  // This constant is defined as null at AppConstants.
```

Comment:
maybe explain this a little more?

Reply:
we're just assigning one constant to another it's value is not changin, I think this should be fine